src/views/Farms/components/SearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useRef } from 'react'
+import { Input } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const StyledInput = styled(Input)`
+  border-radius: 16px;
+  transition: 0.3s;
+  margin-left: auto;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    width: 100% !important;
+    padding: 0 1rem !important;
+    opacity: 1 !important;
```

Comment:
Please double check why `!important` is necessary. In principal if we need an `!important` on our own code then we did something wrong to begin with.